src/aead/aes_gcm.rs

```diff
@@ -52,20 +54,30 @@ fn aes_gcm_seal(ctx: &[u64; aead::KEY_CTX_BUF_ELEMS],
                 -> Result<(), error::Unspecified> {
     let ctx = polyfill::slice::u64_as_u8(ctx);
     bssl::map_result(unsafe {
-        GFp_aes_gcm_seal(ctx.as_ptr(), in_out.as_mut_ptr(), in_out.len(), tag,
-                         nonce, ad.as_ptr(), ad.len())
+        GFp_aes_gcm_seal(ctx.as_ptr(),
+                         in_out.as_mut_ptr(),
+                         in_out.len(),
+                         tag,
+                         nonce,
+                         ad.as_ptr(),
+                         ad.len())
     })
 }
 
 fn aes_gcm_open(ctx: &[u64; aead::KEY_CTX_BUF_ELEMS],
                 nonce: &[u8; aead::NONCE_LEN], in_out: &mut [u8],
                 in_prefix_len: usize, tag_out: &mut [u8; aead::TAG_LEN],
-                ad: &[u8]) -> Result<(), error::Unspecified> {
+                ad: &[u8])
+                -> Result<(), error::Unspecified> {
```

Comment:
This is not such a big deal as the function call formatting, but still it seems unnecessary.